### PR TITLE
Add Basic Screens - closes #1

### LINKED ISSUE #1: Frontend hello world
# Todo

- [x] Add Create React App basics to repository
- [x] Block out components from Figma into flats with sensible splits, minus styling
- [x] Add outline button component

### PR DESCRIPTION
**Pull request by conatus**
Adds basic screens as per Figma design, with hierachy but minimal styling and no functionality.

### CODE REVIEW

File: frontend/src/components/DemandBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+
+import Emoji from "a11y-react-emoji";
+
+type DemandBlockProps = {
+  demand: string;
+};
+
+Emoji;
```
Comment (janbaykara) [inline frontend/src/components/DemandBlock.tsx]: What is this line for?

File: frontend/src/pages/ResultsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    </h2>
+    <hr />
+    <StatisticBlock
+      render={() => (
+        <Fragment>
+          The rent on a typical two-bed home in Tottenham is{" "}
+          <strong>£1,525</strong>. That’s <strong>£850</strong> more than the
+          national average.
+        </Fragment>
+      )}
```
Comment (janbaykara) [inline frontend/src/pages/ResultsPage.tsx]: Does this ned to be an anonymous function? Is this to defer computation to when React deems it necessary?

File: frontend/src/components/DemandBlock.tsx (hunk 1 above)
Comment (conatus) [reply frontend/src/components/DemandBlock.tsx]: My mistake, removed.

File: frontend/src/pages/ResultsPage.tsx (hunk 2 above)
Comment (conatus) [reply frontend/src/pages/ResultsPage.tsx]: Mayee not, but I wanted to enable one to be able to specify the text - perhaps `children` would be more correct. I will refactor once I actually build this out.

**Pull request merged**